app/services/targets/update_service.rb

```diff
@@ -34,18 +47,23 @@ def execute(target_params)
     private
 
     def handle_change_of_evaluation_criteria(evaluation_criteria_ids)
-      if @target.evaluation_criterion_ids.blank? && evaluation_criteria_ids.present?
+      if @target.evaluation_criterion_ids.blank? &&
+           evaluation_criteria_ids.present?
         # Clear submissions without grades when target changes from auto-verified to evaluated.
-        TimelineEvent.left_joins(:timeline_event_grades)
-          .where(target: @target, timeline_event_grades: { id: nil })
+        @target
+          .timeline_events
+          .where
+          .missing(:timeline_event_grades)
```

Comment:
This looks interesting! 😄

Reply:
@bodhish Yeah, I'd meant to point this out and forgot - it was a new suggestion from the upgraded Rubocop. Way better than the manual left_join we used to do. 🙂 

@mahesh-krishnakumar Note!